screens/questions/question.css

```diff
@@ -0,0 +1,20 @@
+.question-component {
+  width: fit-content;
+  margin: 3rem auto;
+  padding: 2rem;
+}
+
+.list-stacked > .list-item {
+  margin: 1rem auto;
+  width: 21rem;
+  max-width: 30rem;
+}
+.question-container {
+  margin-top: 1rem;
+}
+
+.question-btn-container {
+  display: flex;
+  justify-content: space-between;
```

Comment:
You can use these as utility classes and import them from the component library. This way you won't be repeating code.

Reply:
I have used utility classes mostly, but this is specifically used for this one only and not for other components,